SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -84,17 +84,11 @@ void USpatialReceiver::LeaveCriticalSection()
 		HandleActorAuthority(PendingAuthorityChange);
 	}
 
-	for (Worker_EntityId& PendingRemoveEntity : PendingRemoveEntities)
-	{
-		RemoveActor(PendingRemoveEntity);
```

Comment:
Why are we no longer removing actors here? Has it been moved elsewhere

Reply:
Discussed offline

Reply:
For context, PendingRemoveEntities wasn't used, remove ops don't come down in critical sections.